cmd/fleetctl/gitops-migrate/README.md

```diff
@@ -0,0 +1,183 @@
+# Overview
+
+This directory contains the `gitops-migrate` tool, designed leading up to the `4.74` Fleet release to automate necessary GitOps YAML transformations.
```

Comment:
Should this live under fleetctl? typically this is reserved for subcommands to fleetctl no?

Wouldn't this be better in `tools` or `cmd/gitops-migrate`?